.circleci/config.yml

```diff
@@ -169,7 +169,24 @@ jobs:
             cd tasks/sep/base-001-MCP-L1
             export SIMULATE_WITHOUT_LEDGER=1
             just \
-              --dotenv-path .env \
+              --dotenv-path $(pwd)/.env \
+              --justfile ../../../single.just \
+              simulate
+
+  just_simulate_eth-011-proto-ver+ownership-transfers:
```

Comment:
CI fails with
```
[#/jobs/just_simulate_eth-011-proto-ver+ownership-transfers] string [just_simulate_eth-011-proto-ver+ownership-transfers] does not match pattern ^[A-Za-z][A-Za-z\s\d_-]*$
ERROR IN CONFIG FILE:
```
so we will need to rename this directory/task to remove the `+`